duckduckgo/macos-browser
# Fix making notarized builds with Network Protection in GitHub Actions

Task/Issue URL: https://app.asana.com/0/1203301625297703/1204688843469939/f
CC: @samsymons 

**Description**:
Add NetP provisioning profiles to build_notarized.yml and import them into keychain before building the app.

**Steps to test this PR**:
Verify that notarized app workflow runs work fine when triggered from this branch:
* [review build](https://github.com/duckduckgo/macos-browser/actions/runs/5095212500/jobs/9159918854)
* [release build](https://github.com/duckduckgo/macos-browser/actions/runs/5095232936/jobs/9159963900)



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
